torch/linalg/__init__.py

```diff
@@ -84,6 +84,54 @@
     True
 """)
 
+cholesky_ex = _add_docstr(_linalg.linalg_cholesky_ex, r"""
+linalg.cholesky_ex(input, *, check_errors=False, out=None) -> (Tensor, Tensor)
+
+Computes the Cholesky decomposition of a complex Hermitian or real symmetric positive-definite matrix.
+
+Supports inputs of float, double, cfloat and cdouble dtypes.
+Also supports batched inputs, and, if the input is batched, the output is batched with the same dimensions.
+
+Returns a namedtuple ``(L,info)``. ``L`` contains the result of the Cholesky decomposition.
+``info`` stores the LAPACK error codes.
+
+If :attr:`input` is not a Hermitian positive-definite matrix, or if it's a batch of matrices
+and one or more of them is not a Hermitian positive-definite matrix,
+then ``info`` stores a positive integer for the corresponding matrix.
+The positive integer indicates the order of the leading minor that is not positive-definite,
+and the decomposition could not be completed.
+``info`` filled with zeros indicates that the decomposition was successful.
+If ``check_errors=True`` and ``info`` contains positive integers, then a RuntimeError is thrown.
+
+.. note:: If ``check_errors=True`` given inputs on a CUDA device, this function synchronizes that device with the CPU.
```

Comment:
Users may concern what could happen with `check_errors=False`. Do we also want to mention that behavior? Like, *the function may synchronize that device with the CPU*, without giving too many details?